clients/cobol-lsp-vscode-extension/src/services/LanguageClientService.ts

```diff
@@ -122,21 +123,25 @@ export class LanguageClientService {
         LANGUAGE_ID,
         "LSP extension for " + LANGUAGE_ID.toUpperCase() + " language",
         this.createServerOptions(this.executablePath)!,
-        this.createClientOptions(),
+        this.createClientOptions(vscode.Uri.file(Utils.getExtensionFolder())),
```

Comment:
just change it to a constructor private parameter/member and pass it in at object creation.

Reply:
resolved at : 19c31007cf0f1c59dafd17843eeb8d6bae0c4e67